index.html

```diff
@@ -683,6 +683,284 @@ <h4>Proof Verification (ecdsa-2019)</h4>
 algorithm Elliptic Curve Digital Signature Algorithm (ECDSA) [[FIPS-186-5]],
 with <var>hashData</var> as the data to be verified against the
 <var>proofBytes</var> using the public key specified by
+<var>publicKeyBytes</var>.
+            </li>
+            <li>
+Return <var>verificationResult</var> as the <em>verification result</em>.
+            </li>
+          </ol>
+
+        </section>
+      </section>
+      <section>
+        <h3>jcs-ecdsa-2019</h3>
+
+        <p>
+The `jcs-ecdsa-2019` cryptographic suite takes an input document, canonicalizes
+the document using the JSON Canonicalization Scheme [[RFC8785]], and then 
+cryptographically hashes and signs the output
+resulting in the production of a data integrity proof. The algorithms in this
+section also include the verification of such a data integrity proof.
+        </p>
+
+        <section>
+          <h4>Add Proof (jcs-ecdsa-2019)</h4>
```

Comment:
I guess this is really a crypto suite specific thing, and we expect to see this repeated for eddsa ?

Reply:
Its implicitly in [jcs-eddsa-2020](https://w3c.github.io/vc-di-eddsa/#jcs-eddsa-2022), i.e., section 3.2. That section was written as a "diff" and mine as a "full listing" see https://github.com/w3c/vc-di-eddsa/issues/46.

Reply:
I'm pretty concerned about all the copy-paste between the core algorithms. I'd prefer we do diffs until we write code to generate the full listings (so we don't get editors accidentally messing the algorithm up in one place, or forgetting to update the algorithm everywhere).

That said, having the algorithm in the spec is better than not, so not going to block the PR because of that.

Merge conflicts, tho... please fix. :)